src/components/adaptive-cards/hooks/useActionOpenUrl.js

```diff
@@ -0,0 +1,24 @@
+import {isValidUrl} from '../../../util';
+
+/**
```

Comment:
Custom hook that receives an adaptive card OpenURL action and returns html attributes for the target element

Reply:
If no action is received, no attributes are returned